app/helpers/welcome_content_helper.rb

```diff
@@ -0,0 +1,171 @@
+        text: "Ms Colley, my English teacher at school. She left such a strong imprint on me and really instilled a love of reading. I’ll never forget her, she showed me what I could achieve.",
+      },
+    },
+    quote: {
+      headline: "From delving into literature to exploring important life themes, it’s all about taking young minds on a journey as you encourage students to think for themselves. So, get set for some thought-provoking conversations you’ll never forget!",
+      text: "Those moments when kids suddenly get it, where they grasp a concept, that’s when you’re creating change. I can’t think of any other job where you can have that impact.",
+      attribution: "David, English teacher",
+    },
+  }.freeze
+
+  MFL = {
+    video: "welcome-guide-mfl.webm",
+    story: {
+      name: "Tom",
+      job_title_and_location: "Spanish teacher, Coventry",
+      subject: "languages",
+      thumbnail: "dimitra-desert-crop.jpg",
+      image: "dimitra-desert.jpg",
+      alt: "Spanish teacher Tom",
+      text: "Meet Tom - find out how he got into teaching Spanish and what he's looking forward to next.",
+      three_things_youll_never_hear_me_say: {
+        "Every day is the same" => "From exploring Latin American culture to reading Spanish literature, every lesson is different. And with 30 different personalities in one room, it’s never dull",
+        "I miss adult conversation" => "Some of my closest friends are my colleagues, and having someone to talk to makes work life so much easier. Our trips to the pub at the end of the week are something I always look forward to.",
+        "Work never stops" => "It’s important to relax after school,  I can’t resist watching a good Spanish film on Netflix.  And I'm still an avid traveller, I love exploring other cultures.",
+      },
+      shoutout: {
+        name: "Ms Langley and Ms Meredith",
+        relationship: "my Spanish teachers at school",
+        text: "I was totally in awe of my GCSE and A Level Spanish teachers. I loved hearing about their experiences and love of language. It definitely had an impact on my decision to study Spanish at university. We’ve stayed in touch and I still turn to them for advice or ideas when I need. Thank you!",
+      },
+    },
+    quote: {
+      headline: "From delving into different cultures to discussing new ways of thinking, it’s all about taking young minds on a journey out of the classroom. So, get set for some fun and memorable conversations!",
+      text: "There’s a lot of sharing of stories and the kids can be so funny, they really crack me up. Plus, I enjoy the holidays. I get to travel still and can keep up with my languages.",
+      attribution: "Jacqui, modern foreign languages teacher",
+    },
+  }.freeze
+
+  GENERIC = {
+    video: "welcome-guide-generic.webm",
+    story: {
+      name: "Abigail",
+      job_title_and_location: "head of maths, Wigan",
+      subject: nil,
+      thumbnail: "abigail-classroom-crop.jpg",
```

Comment:
Add alt text:

Maths teacher Abigail standing in front of a brightly coloured classroom wall